### PR TITLE
Add more NUI key information

### PR DESCRIPTION
**Pull request by JSUYA**
Pass timestamp and device info to embedder

related pr : https://github.com/flutter-tizen/engine/pull/349

### CODE REVIEW

File: embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs [hunk 1]
```diff
@@ -145,7 +145,7 @@ private void RegisterEventHandlers()
                 }
 
                 FlutterDesktopViewOnKeyEvent(
-                    View, e.Key.KeyPressedName, e.Key.KeyPressed, (uint)e.Key.KeyModifier, (uint)e.Key.KeyCode,
+                    View, e.Key.KeyPressedName, e.Key.KeyPressed, (uint)e.Key.KeyModifier, (uint)e.Key.KeyCode, e.Key.Time,
```
Comment (bbrto21) [inline embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs:148]: I think you may also need below properties, these are may affect input panel behaviors on TV.
Could you please check?
```
DeviceSubClass
DeviceClass
DeviceName
```

Comment (JSUYA) [reply embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs]: Thanky you for check.
I add more key information.

**Review by bbrto21 — APPROVED** [review]

File: embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs [hunk 2]
```diff
@@ -145,7 +145,7 @@ private void RegisterEventHandlers()
                 }
 
                 FlutterDesktopViewOnKeyEvent(
-                    View, e.Key.KeyPressedName, e.Key.KeyPressed, (uint)e.Key.KeyModifier, (uint)e.Key.KeyCode,
+                    View, e.Key.DeviceName, (uint)e.Key.DeviceClass, (uint)e.Key.DeviceSubClass, e.Key.KeyPressedName, e.Key.KeyPressed, (uint)e.Key.KeyModifier, (uint)e.Key.KeyCode, e.Key.Time,
```
Comment (swift-kim) [inline embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs:148]: Exceeds column length limit.

Comment (JSUYA) [reply embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs]: done

**Comment by bbrto21** [comment]
@JSUYA Please resolve conflict.

**Pull request merged**